src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -44,15 +44,15 @@ export class QuestionSubmissionFormComponent implements OnInit {
   set formModel(model: QuestionSubmissionFormModel) {
     this.model = model;
     this.visibilityStateMachine =
-        this.feedbackQuestionsService.getNewVisibilityStateMachine(model.giverType, model.recipientType);
-    const visibilitySetting: {[TKey in VisibilityControl]: FeedbackVisibilityType[]} = {
+      this.feedbackQuestionsService.getNewVisibilityStateMachine(model.giverType, model.recipientType);
+    const visibilitySetting: { [TKey in VisibilityControl]: FeedbackVisibilityType[] } = {
```

Comment:
Just curious. Are you getting lint errors for the original code? Similar for other places with only formatting changes.

Reply:
I don't recall changing these lines as well as the other lines for formatting reasons, shall I try and revert them?

Reply:
Yep would be great if you can revert them